longtask-timing/longtask-in-childiframe-crossorigin.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE HTML>
+<meta charset=utf-8>
+<title>LongTask Timing: long task in nested child iframe</title>
+<body>
+
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<h1>Long Task in Nested Cross-Origin Child Iframe</h1>
+<div id="log"></div>
+<script>
+  async_test(function (t) {
+    var observer = new PerformanceObserver(
+        t.step_func(function (entryList) {
+            var entries = entryList.getEntries();
+            assert_equals(entries.length, 1,
+                "Exactly one entry is expected.");
+            var longtask = entries[0];
+            assert_equals(longtask.entryType, "longtask");
+            assert_equals(longtask.name, "cross-origin-descendant");
+            assert_greater_than(longtask.duration, 50);
+            assert_equals(longtask.startTime, Math.floor(longtask.startTime),
+                "startTime expected to have 1 miillisecond granularity");
+
+            // Assert the TaskAttributionTiming entry in attribution.
+            assert_equals(longtask.attribution.length, 1,
+                "Exactly one attribution entry is expected");
+            var attribution = longtask.attribution[0];
+            assert_equals(attribution.entryType, "taskattribution");
+            assert_equals(attribution.name, "frame");
+            assert_equals(attribution.duration, 0);
+            assert_equals(attribution.startTime, 0);
+            assert_equals(attribution.frameId, "child-iframe-id");
+            assert_equals(attribution.frameName, "child-iframe-name");
+            assert_equals(attribution.frameSrc, "https://wicg.github.io/longtasks/randomslow.html");
```

Comment:
I think web-platform-tests are run in an environment that prevents arbitrary network access, but they [do have a number of different origins available](https://github.com/w3c/web-platform-tests#running-the-tests).  See, for example, [this test](https://github.com/w3c/web-platform-tests/blob/master/html/semantics/embedded-content/the-iframe-element/cross_origin_parentage.html).